python/pyspark/sql/functions.py

```diff
@@ -2038,13 +2038,19 @@ def _wrap_function(sc, func, returnType):
                                   sc.pythonVer, broadcast_vars, sc._javaAccumulator)
 
 
+class PythonUdfType(object):
```

Comment:
Could you also add the descriptions about these three UDF types? 
- NORMAL_UDF: row-based UDFs
- PANDAS_UDF: single-row vectorized UDFs
- PANDAS_GROUPED_UDF: grouped vectorized UDFs

Reply:
Sure, I'll add the descriptions.